chromium_src/content/public/common/url_constants.cc

```diff
@@ -1,10 +1,12 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 #include "../../../../../content/public/common/url_constants.cc"
 
 namespace content {
 const char kBraveUIScheme[] = "brave";
+const char kkBinanceScheme[] = "com.brave.binance";
```

Comment:
Is that extra `k` a typo?

Reply:
yes but my typos are never actual errors because I only copy and paste symbol names lol. Will fix.